src/node_options.cc

```diff
@@ -667,6 +666,13 @@ PerProcessOptionsParser::PerProcessOptionsParser(
             "print V8 command line options",
             &PerProcessOptions::print_v8_help);
 
+#ifdef NODE_REPORT
+AddOption("--report-on-fatalerror",
+            "generate diagnostic report on fatal (internal) errors",
+            &PerProcessOptions::report_on_fatalerror,
+            kAllowedInEnvironment);            
```

Comment:
This is mis-indented (it’s inside a function so it should not be right at the start of the line)